src/verification.rs

```diff
@@ -621,8 +619,8 @@ impl CancelInfo {
 
     /// Get the `CancelCode` that cancelled this verification.
```

Comment:
no longer a `CancelCode`. It would be nice to document what it actually *is*, with an example.

Reply:
done.  also added changelog and tests